pkg/cmd/opts/kustomize.go

```diff
@@ -0,0 +1,54 @@
+package opts
+
+import (
+	"github.com/blang/semver"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/versionstream"
+	"github.com/pkg/errors"
+)
+
+// EnsureKustomize ensures kustomize is installed
+func (o *CommonOptions) EnsureKustomize() error {
+	version, err := o.Kustomize().Version()
+
+	if err == nil && version != "" {
+		// get the stable jx supported version of kustomize to be install
```

Comment:
maybe we can extract something like a _installedBinaryInSupportedVersionRange_? There might be even already something like this.